satellite/satellitedb/certdb.go

```diff
@@ -18,40 +19,47 @@ type certDB struct {
 	db *dbx.DB
 }
 
-func (b *certDB) SavePublicKey(ctx context.Context, nodeID storj.NodeID, publicKey crypto.PublicKey) error {
-	tx, err := b.db.Open(ctx)
+func (certs *certDB) SavePublicKey(ctx context.Context, nodeID storj.NodeID, publicKey crypto.PublicKey) error {
+	_, err := certs.db.Get_CertRecord_By_Id(ctx, dbx.CertRecord_Id(nodeID.Bytes()))
+	if err == sql.ErrNoRows {
```

Comment:
dbx error wrapping. probably need some NoRows helper

Reply:
Currently it doesn't wrap sql.ErrNoRows

Reply:
No, because it doesn't help here.